android/java/res/xml/quick_action_search_and_bookmark_widget_info.xml

```diff
@@ -8,6 +8,7 @@
     android:initialLayout="@layout/quick_action_search_and_bookmark_widget_layout"
     android:minWidth="304dp"
     android:minHeight="44dp"
+    android:maxResizeHeight="270dp"
```

Comment:
added the max resize height otherwise it could be expanded on the whole home screen and we have an empty space only after row 3.